pyth-sdk/README.md

```diff
@@ -81,3 +81,49 @@ println!("0.1 BTC and 0.05 ETH are worth: ({} +- {}) x 10^{} USD",
 ```
 
 This operation can be useful for pricing, e.g., an LP token that is backed by two underlying currencies.
+
+## Liquidity-related Pricing Measures
+
+A variety of measures should be taken for protocols looking to mitigate liquidity-related risks. To limit deposits and borrows, protocols could simply derive their limit from the off-chain liquidity estimates and then store those values in contract state. No explicit interaction with the Pyth SDK would be needed in this simple case.
```

Comment:
cut the paragraph above -- the README should describe stuff that is in this code, and it needs to be short and to the point.